### PR TITLE
HTML і CSS практика: Hooli-style Popup

### PR DESCRIPTION
**Pull request by loki13btz**
### HTML і CSS практика: Hooli-style Popup

[Demo](https://loki13btz.github.io/Hooli-style-Popup/) |
[Code base](https://github.com/loki13btz/Hooli-style-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.

**Comment by loki13btz** [comment]
@A-Ostrovnyy PR updated, take a look please)

File: submissions/loki13btz/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Hooli-style Popup</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="header-navigation">
+            <ul class="menu-list">
+                <li><a href="#" class="menu-item">Mail</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/loki13btz/html-css-popup/index.html:16]: Avoid links jumping on hover [watch video](https://share.vidyard.com/watch/hYhiY3ma8czAXmnR73ypa6?)

File: submissions/loki13btz/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                <li><a href="#" class="menu-item">Images</a></li>
+                <li><label for="popupCheckbox" class="menu-item popup-shower"><img class="popup-icon"
```
Comment (A-Ostrovnyy) [inline submissions/loki13btz/html-css-popup/index.html:18]: Add `cursor: pointer` to all interactive elements.
Also, the user must have the ability to open a popup using keyboard.

File: submissions/loki13btz/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                            <p>Promo</p>
+                        </a></li>
+
+                    <li><a href="#!" class="content-item">
+                            <img src="./img/search.png" alt="" class="content-img">
+                            <p>Search</p>
+                        </a></li>
+                </ul>
+
+                <label for="buttonCheckbox" class="open-button">More</label>
```
Comment (A-Ostrovnyy) [inline submissions/loki13btz/html-css-popup/index.html:114]: Add styles on hover and focus states.
Also, add `cursor: pointer`

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by loki13btz** [comment]
@A-Ostrovnyy PR updated, take a look please)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**